scripts/final-checks.sh

```diff
@@ -5,7 +5,8 @@
 
 set -e
 
-cargo clippy --workspace -- -D warnings
+cargo +nightly clippy --lib --bins --tests --examples --workspace -- -D warnings
+cargo fmt
 cargo fmt --all -- --check
```

Comment:
why both `cargo fmt` and `cargo fmt --all -- --check`?

Reply:
True, I'll remove the check, just keep `cargo fmt --all`